CONTRIBUTING.md

```diff
@@ -18,19 +45,27 @@ by running:
 openrct2 --version
 ```
 
-For Windows builds, OpenRCT2 will generate a memory dump and saved game when the game crashes unexpectedly. The game
-will open explorer to these files automatically for you. They are placed inside your configured user directory which
-by default is `%HOMEPATH%\Documents\OpenRCT2`.
+For Windows builds, OpenRCT2 will generate a memory dump and saved game when the game crashes unexpectedly. The game will open explorer to these files automatically for you. They are placed inside your configured user directory which by default is `%HOMEPATH%\Documents\OpenRCT2`.
+
+
+## 4.3 Fixing bugs
+A list of bugs can be found on the [issue tracker](https://github.com/OpenRCT2/OpenRCT2/issues). Feel free to work on any bug and submit a pull request to the develop branch with the fix. Mentioning that you intend to fix a bug on the issue will prevent other people from trying as well.
+
+## 4.4 Translation
+You can translate the game into other languages by editing the language files in ```data/language``` directory. Please join discussions in the [#localisation channel on Discord](https://discordapp.com/invite/sxnrvX9) and submit pull requests to [OpenRCT2/Localisation](https://github.com/OpenRCT2/Localisation).
+
+## 4.5 Graphics
+You can help create new graphics for the game by visiting the [OpenGraphics project](https://github.com/OpenRCT2/OpenGraphics). 3D modellers needed!
+
+## 4.6 Audio
+You can help create the music and sound effects for the game. Check out the OpenMusic repository and drop by our [#open-sound-and-music channel on Discord](https://discord.gg/9y8WbcX) to find out more.
 
-# Translation
-Translation is managed in a separate repository, [OpenRCT2/Localisation](https://github.com/OpenRCT2/Localisation).
-You will find more information there.
+## 4.7 Scenarios
+We would also like to distribute additional scenarios with the game, when the time comes. For that, we need talented scenario makers! Check out the [OpenScenarios repository](https://github.com/PFCKrutonium/OpenRCT2-OpenScenarios).
```

Comment:
This repository has been moved into the organization, so  the link can be updated
```suggestion
We would also like to distribute additional scenarios with the game, when the time comes. For that, we need talented scenario makers! Check out the [OpenScenarios repository](https://github.com/OpenRCT2/OpenScenarios).
```